### ISSUE TITLE
Issue with less files

### ISSUE THREAD

**Issue by amitmtrn** [nirkaufman/angular-snake#1]
Do you have any clue how to fix it?

```
Home@HOME-PC /d/Projects/angular-snake (master)
$ npm start

> angular-snake@1.0.0 start d:\Projects\angular-snake
> webpack-dev-server --content-base src/

http://localhost:8080/webpack-dev-server/
webpack result is served from /
content is served from d:\Projects\angular-snake\src
Hash: 9a157d4ba274d43830a1
Version: webpack 1.9.8
Time: 2790ms
    Asset     Size  Chunks             Chunk Names
bundle.js  2.56 MB       0  [emitted]  main
chunk    {0} bundle.js (main) 976 kB [rendered]
    [0] ./src/app/app.js 809 bytes {0} [built]
    [1] ./~/angular/angular.js 960 kB {0} [built]
    [2] ./src/navbar/navbar.js 1.49 kB {0} [built]
    [3] ./src/scores/scores.js 1.86 kB {0} [built]
    [4] ./~/angular/index.js 48 bytes {0} [built]
    [5] ./src/app/app.less 1.04 kB {0} [built] [2 errors]
    [6] ./src/app/app.html 245 bytes {0} [built]
    [7] ./src/snake/snake.js 6.12 kB {0} [built]
    [8] ./src/snake/snake.less 1.05 kB {0} [built] [2 errors]
    [9] ./src/snake/snake.html 176 bytes {0} [built]
   [10] ./src/scores/scores.html 362 bytes {0} [built]
   [11] ./src/common/storage.js 1.27 kB {0} [built]
   [12] ./src/navbar/navbar.less 1.05 kB {0} [built] [2 errors]
   [13] ./src/navbar/navbar.html 474 bytes {0} [built]
   [14] ./src/snake/config.json 88 bytes {0} [built]

ERROR in ./src/app/app.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/css-loader/index.js in d:\Projects\angular-snake/src\app
 @ ./src/app/app.less 4:14-120

ERROR in ./src/app/app.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/style-loader/addStyles.js in d:\Projects\angular-snake/src\app
 @ ./src/app/app.less 7:13-71

ERROR in ./src/snake/snake.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/css-loader/index.js in d:\Projects\angular-snake/src\snake
 @ ./src/snake/snake.less 4:14-122

ERROR in ./src/snake/snake.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/style-loader/addStyles.js in d:\Projects\angular-snake/src\snake
 @ ./src/snake/snake.less 7:13-71

ERROR in ./src/navbar/navbar.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/css-loader/index.js in d:\Projects\angular-snake/src\navbar
 @ ./src/navbar/navbar.less 4:14-123

ERROR in ./src/navbar/navbar.less
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modul
es/style-loader/addStyles.js in d:\Projects\angular-snake/src\navbar
 @ ./src/navbar/navbar.less 7:13-71
webpack: bundle is now VALID.

```

**Comment by eyalap** [comment]
The problem is that your __dirname variable is with backslashes instead of slashes (windows right ?)
To make this demo work you could simply replace resolve: { root...} with 
 root: [__dirname.replace(/\\/g, "/") + '/src']
I hope someone else will have a better solution for this

**Comment by amitmtrn** [comment]
Fixed, you can close the issue.
I copied the fix from pull request #2
